regression/run/struct/python/test.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+# other Shroud Project Developers.
+# See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#
+# #######################################################################
+#
+# test struct.yaml
+#
+from __future__ import print_function
+
+import numpy as np
+import unittest
+import cstruct
+
+class Struct(unittest.TestCase):
+    """Test struct problem"""
+     
+    def XXsetUp(self):
+        """ Setting up for the test """
+        print("FooTest:setUp_:begin")
+        ## do something...
+        print("FooTest:setUp_:end")
+     
+    def XXtearDown(self):
+        """Cleaning up after the test"""
+        print("FooTest:tearDown_:begin")
+        ## do something...
+        print("FooTest:tearDown_:end")
+
+    def test_dtype(self):
```

Comment:
@ltaylor16 -- it is a bit difficult to know where to look on this PR.

It seems like this file exercises the generated API from python.

Reply:
The tests generates a lot of code that is hard to look at.  I'm not yet of fan of how github deals with pull requests.  But you're correct, looking at the struct/python/test.py file demonstrates how the users sees the module so it's a good summary of changes.